stl/inc/xstring

```diff
@@ -2195,6 +2195,14 @@ public:
 template <class _Ty>
 constexpr size_t _Size_after_ebco_v = is_empty_v<_Ty> ? 0 : sizeof(_Ty); // get _Ty's size after being EBCO'd
 
+struct _String_constructor_concat_tag {

```

Comment:
I'd like a comment saying what this tag does.

Something like: 
```
// used in operator+ when we've already computed the size, to avoid another strlen check
```
(not sure if that's a 100% accurate description of what it does)